account_billing/report/report.xml

```diff
@@ -1,11 +1,12 @@
 <odoo>
-    <report
-        id="report_account_billing"
-        model="account.billing"
-        string="Billing"
-        report_type="qweb-pdf"
-        name="account_billing.report_billing"
-        file="account_billing.report_billing"
-        print_report_name="'BILL-%s%s' % (object.name, object.state == 'draft' and '-draft' or '')"
-    />
+    <record id="report_account_billing" model="ir.actions.report">
+        <field name="name">Billing</field>
+        <field name="model">account.billing</field>
+        <field name="report_name">event.event_registration_report_template_badge</field>
```

Comment:
Why are you change it to
`<field name="report_name">event.event_registration_report_template_badge</field>`

It should be
`<field name="report_name">account_billing.report_billing</field>`

Reply:
@Saran440  Thank you for Code Review. The code has been edited.